src/main/run_work.py

```diff
@@ -19,10 +19,18 @@ def _check_config(path):
 
 
 def _check_draft(path,draft_config):
-    if SaveApiKey().exsitapikey() and not os.path.exists(os.path.join(path, "ta", "draft.json")) and not draft_config:
-        return False
-    elif not SaveApiKey().exsitapikey() and not os.path.exists(os.path.join(path, "ta", "draft.json")):
-        return False
+    # Not have draft.json
+    if not os.path.exists(os.path.join(path, "ta", "draft.json")):
+        # Have an api-key
+        if SaveApiKey().exsitapikey():
+            # Choose read draft
+            if not draft_config:
+                return False
+            # Choose fetch draft
+            return True
+        # Not have an api-key and draft.json
+        if not SaveApiKey().exsitapikey():
```

Comment:
Maybe. this line is  not necessary. Opposite of True is always false and you use the return syntax in the upper condition so there is no chance the code can get here if the apiKey is existed (`SaveApiKey().exsitapikey()` returned true) but if it can it means that the apiKey is not existed.